vignettes/installation.Rmd

```diff
@@ -33,8 +33,15 @@ remotes::install_cran('azuremlsdk', repos = 'https://cloud.r-project.org/', INST
 ```
 
 ## Install the Azure ML Python SDK
-Lastly, use the **azuremlsdk** R library to install the Python SDK. By default, `azuremlsdk::install_azureml()` will install the [latest version of the Python SDK](https://pypi.org/project/azureml-sdk/) in a conda environment called `r-azureml` if reticulate < 1.14 or `r-reticulate` if reticulate ≥ 1.14.
-``` {r install_pythonsdk, eval=FALSE}
+Lastly, use the **azuremlsdk** R library to install the [latest version of the Python SDK](https://pypi.org/project/azureml-sdk/).
+
+If using an Azure Machine Learning Compute Instance or a CRAN installation with `reticulate >= 1.14`, use the `azuremlsdk::install_azureml()` function with the conda environment set to `r-reticulate`.
```

Comment:
if user is on their local machine and reticulate == 1.14, will they still need to specify the envname?

Reply:
instructions here conflict with the one below, which says "If using a CRAN installation with reticulate < 1.14 or a GitHub installation, by default, azuremlsdk::install_azureml() will install in a conda environment called r-azureml if reticulate < 1.14 or r-reticulate if reticulate ≥ 1.14. There is no need to specify the conda environment."

Reply:
Yes, they'll need to specify if using a CRAN installation with reticulate >= 1.14. 
If reticulate < 1.14 and using CRAN, there's no need to specify. Similarly, if using a GitHub installation, there is no need to specify. 

Because the issue is with changes to reticulate 1.14. The patch I made last week to the install script handles that issue, but, of course, that's only reflected in the GitHub repo, not the CRAN release, so if anyone is using a CRAN release with the new reticulate, they'll run into an issue and will need to specify the environment themselves in order to troubleshoot. Does that make sense?

I'll try to word it more clearly.

Reply:
oh you're right - I missed the CRAN part.